article/sections/_2_graph_importance.tex

```diff
@@ -0,0 +1,200 @@
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{WTD_interacting_graph.png}
+   \caption{Causal Graph for the Walk-Transit-Drive Utility Function}
+   \label{fig:WTD_causal_2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{DTW_interacting_graph.png}
+   \caption{Causal Graph for the Drive-Transit-Walk Utility Function}
+   \label{fig:DTW_causal_2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{WALK_interacting_graph.png}
+   \caption{Causal Graph for the Shared Ride 3+ Utility Function}
+   \label{fig:WALK_causal_2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{BIKE_interacting_graph.png}
+   \caption{Causal Graph for the Bike Utility Function}
+   \label{fig:BIKE_causal_2}
+\end{figure}
+
+Using the setup described above, we conduct our simulation exercise as follows:
+\begin{enumerate}
+
+\item We build a DAG (Figure \ref{fig:IND_GRAPH}) where we assume all explanatory variables are independent of each other.
+\item We simulate data based on this graph and generate outcomes based on a chosen ``true'' multinomial logit model.
+\item We predict outcomes based on the mode choice model.
+\item We apply the do-operator \citet{pearl_causality_2000} to our simulated data to reduce the travel distance for all individuals in the dataset. This emulates a company's decision to move its employees closer to campus.
+\item We predict new outcomes based on the mode choice model.\newline
+\item We build a different DAG (Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}) for each utility function based on assumptions of how explanatory variables interact and influence the outcome.
+\item We use the real dataset to estimate the relationships between different variables outlined in Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}.
+\item We simulate data for variables without parent nodes in Figure Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2} and we use the estimated relationships from step 6 to simulate the remaining explanatory variables.
+\item We then use the choice model to simulate outcome choices based on data simulated for Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}.
+\item We apply the do-operator, as above, to reduce the travel distance for all individuals in the dataset and to emulate a company's decision to move its employees closer to campus.
+\item We use the estimated relationships for this causal graph to simulate all the explanatory variables in Figure \ref{fig:DA_causal_2} through Figure \ref{fig:BIKE_causal_2}(namely the variables affected by travel distance).
+\item We use the estimated choice model to produce outcomes based on the simulated data from the previous step.
+
+\end{enumerate}
+
+We repeat this simulation process several times and recover the choice probability of all modes for all individuals.
```

Comment:
We should add a sentence pointing readers to more information.
Specifically, we should link to our code/notebook for performing the simulations.
We should tell readers its where they can see the details of how we chose our distributions for simulating each variable and any other aspects of the simulation they're interested in.

Reply:
Added with reference in bib file. Check and let me know.

Reply:
🙏 thank you!